### ISSUE TITLE
assert scope["type"] == "http"`on websocket connection 

### ISSUE THREAD

**Issue by kvspb** [elastic/apm-agent-python#1186]
`
    ...lib/python3.9/site-packages/elasticapm/contrib/starlette/__init__.py", line 135, in __call__
    assert scope["type"] == "http"
AssertionError
`

**To Reproduce**

```py
import uvicorn
from elasticapm.contrib.starlette import make_apm_client, ElasticAPM
from fastapi import FastAPI, WebSocket

apm = make_apm_client({'SERVICE_NAME': 'test', 'SERVER_URL': 'http://0.0.0.0:8200'})
app = FastAPI()
app.add_middleware(ElasticAPM, client=apm)


@app.websocket('/ws/')
async def websocket_endpoint(
        websocket: WebSocket,
):
    await websocket.accept()


if __name__ == '__main__':
    uvicorn.run(app, host='0.0.0.0', port=18000)

```

run in Chrome console
```js
socket = new WebSocket('ws://127.0.0.1:18000/ws/');
```

**Environment (please complete the following information)**
- OS: MacOs
- Python version: 3.9
- Framework and version: FastAPI
- APM Server version: 3.6.1

**Comment by basepi** [comment]
Thank you for the report! I think instead of using an `assert` we need to just ignore non-http scopes. I'm going to work on this.

**Comment by basepi** [comment]
I have a fix here: https://github.com/basepi/apm-agent-python/tree/asgiscope

Just working on verifying that it fixes both your problem and #1184 before I open the PR. Can you please test if you have a moment?
